### PR TITLE
[CCTRI-2683] - Add skeleton

### PR DESCRIPTION
**Pull request by TarasMaliuchenko**
Add skeleton

### CODE REVIEW

**Review by MaxMhonc — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,18 +1,12 @@
...
-Generic Docker Relay template not bound to any real third-party Cyber Threat
-Intelligence service provider.
+A Cisco SecureX Concrete Relay implementation using [Devo](https://www.devo.com/) as a third-party Cyber Threat Intelligence service provider.
 
-**NOTE.** The template aims to show the general structure for future
-implementations. It also provides a couple of utility functions that might be
-handy. Keep in mind that the main idea here is to just give you a hint of a
-possible approach rather than enforcing you to do everything exactly the same
-way.
+The Relay itself is just a simple application written in Python that can be easily packaged and deployed.
```
Comment (ytsek) [inline README.md:7]: Should we add here the mention of Python version we used?
Something like: "This application was developed and tested under Python version 3.9"

File: code/requirements.txt [hunk 2]
```diff
@@ -1,8 +1,8 @@
 cryptography==3.3.2
-Flask==1.1.2
-marshmallow==3.8.0
-requests==2.24.0
-pyjwt[crypto]==2.0.0
-flake8==3.8.3
+Flask==2.0.0
```
Comment (ytsek) [inline code/requirements.txt:2]: Should we update the Flask version to newest 2.0.1?

**Review by ytsek — DISMISSED** [review]

File: code/requirements.txt (hunk 2 above)
Comment (TarasMaliuchenko) [reply code/requirements.txt]: Updated to the newest - 0ef20f8

**Review by ytsek — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (TarasMaliuchenko) [reply README.md:7]: Mention added - b1bc47e. Thank you.

**Review by ytsek — APPROVED** [review]

**Review by MaxMhonc — APPROVED** [review]

**Pull request merged**